tasks/push-snapshot/tests/test-push-snapshot.yaml

```diff
@@ -78,23 +90,27 @@ spec:
               #!/usr/bin/env sh
               set -eux
 
-              if [ $(cat $(workspaces.data.path)/mock_cosign.txt | wc -l) != 1 ]; then
-                echo Error: cosign was expected to be called 1 times. Actual calls:
+              # 2 for comp1 (the 2 provided tags)
+              # 3 for comp2 (provided tag, once for image, once for source container, + once for source tag)
+              if [ $(cat $(workspaces.data.path)/mock_cosign.txt | wc -l) != 5 ]; then
+                echo Error: cosign was expected to be called 5 times. Actual calls:
                 cat $(workspaces.data.path)/mock_cosign.txt
                 exit 1
               fi
 
-              if [ $(cat $(workspaces.data.path)/mock_skopeo.txt | wc -l) != 2 ]; then
-                echo Error: skopeo was expected to be called 2 times. Actual calls:
+              # 3 for comp1 (same 2 as cosign + 1 for origin digest
+              # 5 for comp2 (same 3 as cosign + 1 for origin digest + 1 for source container digest
+              if [ $(cat $(workspaces.data.path)/mock_skopeo.txt | wc -l) != 8 ]; then
+                echo Error: skopeo was expected to be called 8 times. Actual calls:
                 cat $(workspaces.data.path)/mock_skopeo.txt
                 exit 1
               fi
 
-              test $(jq -r '.images[0].name' $(workspaces.data.path)/results/push-snapshot-results.json) == "comp"
+              test $(jq -r '.images[0].name' $(workspaces.data.path)/results/push-snapshot-results.json) == "comp1"
               test $(jq -r '.images[0].shasum' $(workspaces.data.path)/results/push-snapshot-results.json) == \
-                "sha256:fe05fffad63015c37ef9e0c566638bd5480a5a11c710ebf7c2b91f1a5bc4a3e2"
+                "sha256:5134be780694b03e072a1735d0eea03b6e31ae265460a0b46b50424d46686f3f"
```

Comment:
how did you get this value? I tried getting  the `sha256sum`  of the `docker://registry.io/image1:tag1` but it did not match. I might read the mock not properly. Can you please clarify?

Reply:
I think it comes from https://github.com/konflux-ci/release-service-catalog/blob/development/tasks/push-snapshot/tests/mocks.sh#L40. TBH, I just let the test fail and saw what the value was then put it in the file

Reply:
Found. It keeps the `\n` to calculate the hash.